tools/extractCiphersuites.go

```diff
@@ -0,0 +1,68 @@
+package main
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"os/exec"
+	"strconv"
+	"strings"
+)
+
+type Cipher struct {
+	// Name  string     `json:"name"`
+	Proto string     `json:"proto"`
+	Kx    string     `json:"kx"`
+	Au    string     `json:"au"`
+	Enc   Encryption `json:"encryption"`
+	Mac   string     `json:"mac"`
+}
+
+type Encryption struct {
+	Cipher string `json:"cipher"`
+	Bits   int    `json:"key"`
+}
+
+var opensslpath = "openssl"
```

Comment:
I think we should use the one in cipherscan rather than the system one. It will support more ciphers.